### ISSUE TITLE
Need help with javascript for form

### ISSUE THREAD

**Issue by hldouglas** [hldouglas/dog-rescue-pattern-library#4]
1. When lawyer is checked yes, text box needs appear. When lawyer is checked no the text doesn't need to appear and it isn't required.

2. Attempted to recreated other with text box radio button but the text box is always there.

**Comment by hldouglas** [comment]
@thomasjbradley

**Comment by thomasjbradley** [comment]
I don't really want to just do it for you without you around.

What have you attempted so far with the radio-button/CSS solution?

**Comment by hldouglas** [comment]
I attempted to use the same method we did for the relationship with children section by using different names and adding it to the script at the bottom. I added it to the same script from before, but I don't know if javascript will read the function like that.
For the purposes of the assignment, is it all right if the attempt is there but it didn't work? I didn't mean to make my form so complicated.

**Comment by thomasjbradley** [comment]
Yep, it's completely okay. Implementing any Javascript functionality was never required.

**Comment by hldouglas** [comment]
Probably will be asking questions over the summer then as my client does want her website to go live asap.

**Comment by thomasjbradley** [comment]
Sure thing.
